go/common/node/tee.go

```diff
@@ -27,6 +27,9 @@ type TEEFeaturesSGX struct {
 
 	// DefaultMaxAttestationAge is the default maximum attestation age (in blocks).
 	DefaultMaxAttestationAge uint64 `json:"max_attestation_age,omitempty"`
+
+	// TDX is a feature flag specifying whether support for TDX is enabled.
+	TDX bool `json:"tdx,omitempty"`
```

Comment:
I would rather not because TEE features are already available in the place where this needs to be verified and getting `FeatureVersion` there would require more changes.